core/admin/user.py

```diff
@@ -1,6 +1,62 @@
+from django.conf import settings
 from django.contrib import admin
-from django.contrib.auth.admin import UserAdmin
+from django.contrib import messages
+from django.contrib.auth.forms import PasswordResetForm
 
 from core.models.user import User
 
-admin.site.register(User, UserAdmin)
+
+@admin.register(User)
+class UserAdmin(admin.ModelAdmin):
+    list_display = [
+        "email",
+        "first_name",
+        "last_name",
+        "username",
+        "is_staff",
+        "is_active",
+    ]
+
+    fields = (
+        "email",
+        "first_name",
+        "last_name",
+        "username",
+        "is_staff",
+        "is_active",
+        "last_login",
+        "date_joined",
+    )
+
+    search_fields = (
+        "email",
+        "username",
+        "first_name",
+        "last_name",
+    )
+
+    readonly_fields = (
+        "last_login",
+        "date_joined",
+    )
+    ordering = ("email",)
+
+    def save_model(self, request, obj, form, change):
+        if not change:
```

Comment:
It would be nice to have a new button called "Reset password" next to Save, Save and add another, Save and continue editing (only enabled when editing an existing user)

Reply:
We should add a task for this and maybe add an action for password reset in user list too 🤔